cvmfs/sync_union_tarball.cc

```diff
@@ -199,6 +164,74 @@ void SyncUnionTarball::Traverse() {
   }
 }
 
+void SyncUnionTarball::ProcessArchiveEntry(struct archive_entry *entry) {
+  std::string archive_file_path(archive_entry_pathname(entry));
+  std::string complete_path =
+      MakeCanonicalPath(base_directory_ + "/" + archive_file_path);
+
+  std::string parent_path;
+  std::string filename;
+  SplitPath(complete_path, &parent_path, &filename);
+
+  CreateDirectories(parent_path);
+
+  SharedPtr<SyncItem> sync_entry = SharedPtr<SyncItem>(new SyncItemTar(
+      parent_path, filename, src, entry, read_archive_signal_, this));
+
+  if (NULL != archive_entry_hardlink(entry)) {
+    const std::string hardlink =
+        base_directory_ + "/" + std::string(archive_entry_hardlink(entry));
+
+    printf("Hardlink %s -> %s\n", hardlink.c_str(), complete_path.c_str());
```

Comment:
This should become a `LogCvmfs` statement, probably with increased verbosity level

Reply:
I am quite keen to just remove this line... Do you think is necessary?

Reply:
Removing is fine, I think.